modules/nw-metallb-example-bgppeer.adoc

```diff
@@ -48,6 +48,11 @@ spec:
   holdTime: "10s"
   bfdProfile: doc-example-bfd-profile-full
 ----
+//Dependency on RHEL bug 2054160 being addressed.Remove note when fixed.
+[NOTE]
+====
+Deleting the bidirectional forwarding detection (BFD) profile and removing the `bfdProfile` added to the border gateway protocol (BGP) peer resource does not disable the BFD. Instead, the BGP peer starts using the default BFD profile. To disable BFD from a BGP peer resource, delete the BGP peer configuration and recreate it without a BFD profile.
```

Comment:
Could we reference https://bugzilla.redhat.com/show_bug.cgi?id=2050824 ?

Reply:
OK was just thinking the same